.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,107 @@
+      value: |
+        ---
+        "### For Assignees (Optional)"
+        If you're working on this bug, use the section below to plan and track your work.
+
+  - type: textarea
+    id: internal_notes
+    attributes:
+      label: Internal Notes (Subtasks, DoD, etc.)
+      description: For your own planning, feel free to use Markdown.
+      placeholder: |
+        - [ ] Investigate root cause 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)